tests/cypress/e2e/form/formSummaryCheck.cy.ts

```diff
@@ -0,0 +1,130 @@
+          if (device === 'desktop') {
+            cy.get('[data-cy=stepper-desktop] [data-cy=stepper-step-6]').click();
+          } else {
+            cy.dataCy('stepper-dropdown').click()
+            cy.get('[data-cy=stepper-mobile] [data-cy=stepper-step-6]').click();
+          }
+          cy.dataCy('form-container').should('be.visible').matchImage()
+        })
+
+        it('6. Checking alert visibility.', () => {
+          cy.dataCy('alert-container').should('exist').should('be.visible')
+        })
+
+        it('7. Checking filled in information are saved', () => {
+          cy.dataCy('form-container').then((form) => {
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).first()).find('[data-cy=summary-row-icon]').should('have.not.class', 'text-error')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(2)).should('contain', this.fileData.name)
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(3)).should('contain', this.fileData.address)
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(4)).should('contain', this.fileData.city)
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(5)).should('contain', this.fileData.zip_code)
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(6)).should('contain', this.fileData.email_wrong)
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(7)).should('contain', this.fileData.phone_number_wrong)
+          })
+        })
+
+        it('8. Checking email & phone validation.', () => {
+          cy.dataCy('form-container').then((form) => {
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(6)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(7)).should('have.class', 'border-red-500')
+          })
+        })
+
+        it('9. Checking empty inputs validation', () => {
+          cy.dataCy('form-container').then((form) => {
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(9)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(10)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(11)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(12)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(13)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(14)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(16)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(18)).should('have.class', 'border-red-500')
+
+            cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(19)).should('have.class', 'border-red-500')
+          })
```

Comment:
Osobne by som skôr použil niečo takéto, menej kódu ale viac priehladné. Aktuálne než sa vyrieši ten problém s undefined by som využil túto možnosť.
```suggestion
cy.dataCy('form-container').then((form) => {
  const rowsToCheck = [9, 10, 11, 12, 13, 14, 16, 18, 19];

  rowsToCheck.forEach((rowIndex) => {
    cy.wrap(Cypress.$('[data-cy=summary-row]', form).eq(rowIndex)).should('have.class', 'border-red-500');
  });
});
```